saleor/discount/migrations/0067_orderdiscount_voucher_to_voucher_code_migration.py

```diff
@@ -0,0 +1,92 @@
+from django.db import migrations
+from django.db.models import Exists, OuterRef
+
+
+# For batch size 1000 with 10_000 per model OrderDiscount/Voucher/VoucherCode objects
+# OrderDiscount Migration took 2.39 seconds.
+# OrderDiscount Memory usage increased by 15.65 MiB.
+# OrderLineDiscount Migration took 2.48 seconds.
+# OrderLineDiscount Memory usage increased by 4.61 MiB.
+# CheckoutLineDiscount Migration took 2.26 seconds.
+# CheckoutLineDiscount Memory usage increased by 3.23 MiB.
+
+# For batch size 1000 with 100_000 per model OrderDiscount/Voucher/VoucherCode objects
+# OrderDiscount Migration took 28.68 seconds.
+# OrderDiscount Memory usage increased by 163.00 MiB.
+# OrderLineDiscount Migration took 27.98 seconds.
+# OrderLineDiscount Memory usage increased by 47.99 MiB.
+# CheckoutLineDiscount Migration took 28.12 seconds.
+# CheckoutLineDiscount Memory usage increased by 35.52 MiB.
+
+
+BATCH_SIZE = 1000
```

Comment:
I am not sure if this is a proper size o the batch. According to docs: https://docs.saleor.io/docs/3.x/developer/community/zero-downtime-migrations#solution, single batch shouldn't take more then one second.

Reply:
zero downtown migration will be done in different ticket, those times are for whole migration.